PlexRequests.UI/Views/Search/Index.cshtml

```diff
@@ -168,14 +168,14 @@
                 {{/if_eq}}
                 {{#if_eq type "tv"}}
                 {{#if available}}
-                <span class="label label-success">Available</span>
+                <span class="label label-success">Available on Plex</span>
                 {{else}}
                 {{#if approved}}
                 <span class="label label-info">Processing request</span>
                 {{else if requested}}
                 <span class="label label-warning">Pending approval</span>
                 {{else}}                
-                <span class="label label-danger">Not Available</span>
+                <span class="label label-danger">Not Requested yet</span>
```

Comment:
@tidusjar i just wondering why. Ofcorse its not avaiable on plex because its not requested yet.
I think its more friendly to have a tag "not requested yet then not avaiable on plex.